packages/request-client.js/src/http-request-network.ts

```diff
@@ -41,8 +41,7 @@ export default class HttpRequestNetwork extends RequestNetwork {
       nodeConnectionConfig?: Partial<NodeConnectionConfig>;
       signatureProvider?: SignatureProviderTypes.ISignatureProvider;
       useMockStorage?: boolean;
-      currencies?: CurrencyInput[];
```

Comment:
Breaking: I suggest to remove this parameter as it's redundant with `currencyManager`

Reply:
between the two? I would say yes: it enables injecting your own implementation, and the CurrencyManager contains more than just a list of currencies. For instance, it contains the list of Aggregators for conversion